features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/SessionReplayConfiguration.kt

```diff
@@ -24,12 +23,15 @@ data class SessionReplayConfiguration internal constructor(
 
     /**
      * A Builder class for a [SessionReplayConfiguration].
+     * @param sampleRate must be a value between 0 and 100. A value of 0
+     * means no session will be recorded, 100 means all sessions will be recorded.
+     * The default value for the sample rate will be 0 meaning that no session replay will be
+     * recorded if the sample rate will not be explicitly set.
      */
-    class Builder {
+    class Builder(private val sampleRate: Float) {
```

Comment:
let's add `@FloatRange(from = 0.0, to = 100.0)` annotation as well here, like it was in the `setSessionReplaySampleRate` method.